From 7e4b8f7772e8bee76d240238fc528ffd0eef0de1 Mon Sep 17 00:00:00 2001
From: toship-botsync <toship.s@botsync.co>
Date: Thu, 24 Dec 2020 17:41:14 +0530
Subject: [PATCH] Added install configuration in CMakeLists.txt

Upstream-Status: Backport [https://github.com/botsync/volta/commit/4cc0ea585c2903865f6f098645746dc4f26c986d]

Adjusted for older version which doesn't have:
https://github.com/botsync/volta/commit/57ecf986699ef7fac8345938b30e9856313f0ea9
so instead of udev_rules.rules there are imu.rules mcu.rules 10-volta-usb.rules

Signed-off-by: Martin Jansa <martin.jansa@lge.com>
---
 CMakeLists.txt | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a4aba61..8a0bcdf 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4,3 +4,16 @@ project(volta_rules)
 find_package(catkin REQUIRED COMPONENTS
 
 )
+
+catkin_package(
+  CATKIN_DEPENDS
+)
+
+install(PROGRAMS
+        create_udev_rules
+  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
+
+install(FILES imu.rules mcu.rules 10-volta-usb.rules
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+)
